tests/crc32/main.c

```diff
@@ -20,6 +24,65 @@ static struct testcase {
 	    {0x1b, 0xc4, 0xb4, 0x28}}
 };
 
+/* Largest buffer must be first. */
+static const size_t perfsizes[] = {16384, 8192, 4096, 2048, 1024, 512, 256,
```

Comment:
Should we add some more tests?  Or repeat these a few times?  it's hard to measure things that take less than 100 us.